deploy/images/calico-node/apko.tmpl.yaml

```diff
@@ -0,0 +1,38 @@
+contents:
```

Comment:
add a comment at the top with a link to the source

https://github.com/chainguard-images/images/blob/4317e36114e989a8dcf744c4685cd60eb8582447/images/calico/configs/latest.node.apko.yaml